### ISSUE TITLE
Merging breaks position_ids in CLIP model

### ISSUE THREAD

**Issue by zwishenzug** [ogkalu2/Merge-Stable-Diffusion-models-without-distortion#29]
Even though the position_ids are supposed to be ignored I think the code still converts them to floats/halfs which they shouldn't be, they should be int64.

A quick fix I did was to put this line before the torch.save:

    theta_0['cond_stage_model.transformer.text_model.embeddings.position_ids'] = torch.tensor([list(range(77))], dtype=torch.int64, device="cpu")

**Comment by WASasquatch** [comment]
I wonder if this is why the saved model differed from the merged model that I used prior. The same fixed seed I was using totally changed and the prompt started making very close up stuff (there was "close up"), but previous tons of diffusion I did before loading the saved model were not as zoomed in and coming out great. I'm also not sure what CLIP I *should* be adding to the save node, especially when merging more than 1 model. I guess that's trial and error like I did before saving but not sure what's up with the loaded model output difference.
